### PR TITLE
Fix a typo in a comment

### PR DESCRIPTION
**Pull request by 0mp**

### CODE REVIEW

File: bin/auditdistd/proto_tls.c [hunk 1]
```diff
@@ -413,7 +413,7 @@ tls_exec_client(const char *user, int startfd, const char *srcaddr,
 	tls_certificate_verify(ssl, fingerprint);
 
 	/*
-	 * The following byte is send to make proto_connect_wait() to work.
+	 * The following byte is sent to make proto_connect_wait() to work.
```
Comment (cbrueffer) [inline bin/auditdistd/proto_tls.c]: Looks good; while there the last "to" should go too.

Comment (0mp) [reply bin/auditdistd/proto_tls.c]: True. I missed that. I'll fix that and squash those two commits.

Comment (0mp) [reply bin/auditdistd/proto_tls.c]: @cbrueffer Aaaand it's fixed. Thank you :+1:

**Comment by cbrueffer** [comment]
Merged, thanks!

**Pull request merged**